clang/test/dpct/cuda_const_usm.cu

```diff
@@ -27,6 +27,12 @@ __constant__ float const_angle[360], const_float[NUM_ELEMENTS][num_elements * 2]
 // CHECK: static dpct::constant_memory<sycl::double2, 0> vec_d;
 __constant__ double2 vec_d;
 
+// CHECK: /*
+// CHECK-NEXT: DPCT1120:{{[0-9]+}}: template_a is passed into device functions as extra arguments. If template_a is template variable, the variable initialization, the extra parameter types and the variable name inside device functions may be incorrect. You may need to adjust the code.
```

Comment:
template_a is device constant memory and passed into device functions as extra arguments. As here template_a is template variable, then the declaration of extra parameter in device functions, the variable reference in command group and device functions may be incorrect. You may need to adjust the code.